packages/api/lib/writeRecords/write-granules.js

```diff
@@ -540,7 +541,7 @@ const _writeGranuleRecords = async (params) => {
         // The only values allowed to be updated in the PG granule write under these conditions are
         // currently status, timestamp, updated_at, and created_at, and the associated execution
         // as part of the write chain
-        const limitedUpdateApiGranuleRecord = await translatePostgresGranuleToApiGranule({
+        limitedUpdateApiGranuleRecord = await translatePostgresGranuleToApiGranule({
```

Comment:
Nit: what about `writeConstrainedApiGranule`?

Reply:
I don't have a strong opinion between these two names, so it can be changed. However, I believe this is all going away soon, so might not be worth it. Do you want it to be changed?